controlpanel/frontend/jinja2/parameter-create.html

```diff
@@ -0,0 +1,108 @@
+{% from "input/macro.html" import govukInput %}
+{% from "fieldset/macro.html" import govukFieldset %}
+{% from "radios/macro.html" import govukRadios %}
+
+{% extends "base.html" %}
+
+{% set page_title = "Create a secret parameter for your webapp or airflow task" %}
+{% set legend -%}
+  Create a secret parameter for your webapp or airflow task
+{%- endset %}
+
+{% set page_name = "parameters" %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl">{{ page_title }}</h1>
+
+  {% if form.error %}
+    <div class="error">{{ form.error }}</div>
+  {% endif %}
+
+  <form action="{{ url('create-parameter') }}" method="post">
+    {{ csrf_input }}
+    {% call govukFieldset({ "legend": { "html": legend|safe } }) %}
+      {{
+        govukInput({
+          "label": {
+            "text": "Key",
+            "classes": "govuk-label--m",
+          },
+          "classes": "govuk-!-width-two-thirds",
+          "hint": {
+            "text": '60 chars max, only letters, numbers and underscores'
```

Comment:
Do users know what a parameter key is? Don't know if the label should be more explicit (maybe "Key" is good), or maybe explain in the hit that this is the parameter identifier?

The thing is, as far as I understand this the key is only appended to the rest of the name. Isn't the name or ARN the thing they're using to actually access the value?